test/test-applications/integrations/Samples.WebRequest.NetFramework20/Program.cs

```diff
@@ -18,7 +19,9 @@ public static class Program
 
         private static string Url;
 
-        public static void Main(string[] args)
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
+        public static async Task Main(string[] args)
+#pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
```

Comment:
This one could still use a comment explaining why we made it `async`.

```suggestion
// Since this project references `Datadog.Trace.csproj` with `Private="false"`,
// `Datadog.Trace.dll` is no longer in the bin folder when this project is built.
// With a `void` method, the app tries to load the dll before the profiler is loaded.
// With `async`, the loader has time to run and load `Datadog.Trace` first. It's hacky, but it works.
// See https://github.com/DataDog/dd-trace-dotnet/pull/1726#discussion_r701218376

#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
        public static async Task Main(string[] args)
#pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
```